easybuild/easyconfigs/b/Boost/Boost-1.47.0-goolf-1.4.10-Python-2.7.5.eb

```diff
@@ -0,0 +1,26 @@
+name = 'Boost'
+version = '1.47.0'
+
+homepage = 'http://www.boost.org/'
+description = """Boost provides free peer-reviewed portable C++ source libraries."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
+
+pythonversion = '2.7.5'
+versionsuffix = '-Python-%s' % pythonversion
+
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('Python', pythonversion),
+]
+
+# also build boost_mpi
+boost_mpi = True
+
+osdependencies = ['zlib-devel']
```

Comment:
A proper zlib dep should be included here. We have the feature that deals with this site-specific policy: `--filter-deps` (see http://easybuild.readthedocs.org/en/latest/Manipulating_dependencies.html)